capture/genBitmaps.js

```diff
@@ -69,6 +69,13 @@ function capturePageSelectors(url,scenarios,viewports,bitmaps_reference,bitmaps_
 
 	casper.each(scenarios,function(casper, scenario, scenario_index){
 
+		if (fs.isFile(scenario.cookiesJsonFile)) {
```

Comment:
Please check for property value here eg., if(scenario.cookiesJsonFile && fs.isFile(...